src/invoice/signature.rs

```diff
@@ -131,6 +132,18 @@ impl KeyEntry {
         let sig = key.sign(self.label.as_bytes());
         self.label_signature = Some(base64::encode(sig.to_bytes()));
     }
+    pub fn verify_label(self, key: PublicKey) -> anyhow::Result<()> {
+        match self.label_signature {
+            // If there is no label signature, then I guess we say everything's good.
```

Comment:
Should we `warn!`?

Reply:
I'm not sure. Here's the thing: The signature is both optional and voluntary. It's basically for moving keyrings from one person to another and keeping trust. So a warning seems heavy-handed. When the file is on your localhost, I _think_ it is fine for the user to edit the label on their own. And they should be able to do that without then having to go generate a new crypto signature... since it's really just a local file at that point.

Reply:
For now, I'll `info!()` it, since that seems to split the difference.